site/docs/uk/hosting/vps.md

```diff
@@ -9,30 +9,130 @@ next: false
 
 Віртуальний приватний сервер, відомий переважно як VPS, є віртуальною машиною, яка працює в хмарі, а користувачі мають повний контроль над її системою.
 
-У цьому посібнику ви дізнаєтеся про різні методи запуску вашого бота на VPS, підтримання його в онлайні 24/7, автоматичний запуск при завантаженні вашого VPS і перезапуск при збоях.
+## Оренда сервера
 
-## systemd
+> Щоб слідувати цьому посібнику, спершу вам потрібно орендувати VPS.
+> У цьому першому розділі пояснюється, як це зробити.
```

Comment:
```suggestion
> У цьому розділі пояснюється, як це зробити.
```

We are already here, this clarification is unnecessary.

Reply:
This change must be applied to the original first. I don't think we should do that because we've already allowed translation. Although I broke this rule, there were serious changes. What do you think, @roj1512?

Reply:
t.me/c/1322374388/5608, we just need to make this change in the English version.